api/documentation/controller.py

```diff
@@ -405,6 +414,50 @@ def addSchemes(self):
             }
         )
 
+        self.addComponent(
+            'schema', 'MARCRollContributorsDict', 'dict',
+            {
+                'items': {
+                    'type': 'object',
+                    'properties': {
+                        'MARCRoleCode': {'type': 'string'},
+                        'ContributorRole': {'type': 'string'},
+                        'MARCRoleCode': 'ContributorRole'
+                    }
+                }
+            }
+        )
+        self.addComponent(
+            'schema', 'LanguageCodes', 'dict',
+            {
+                'items': {
+                      'type': 'object',
+                      'properties': {
+                          'language_code': {'type': 'string'},
+                          'languages': {'type': "List['string']"},
+                          'language_code': 'languages'
+                      }
+                }
+            }
+        )
+        self.addComponent(
+            'schema', 'MediaSchemaDict', 'dict',
+            {
+                'items': {
+                      'type': 'object',
+                      'properties': {
+                          'href': {'type': 'string'},
+                          'media_type': {'type': 'string'},
+                          'href': 'media_type'
+                      }
+                }
+            }
+        )
+        self.addComponent(
+            'schema', 'BulkCirculation', 'list',
```

Comment:
Should be `array` here, instead of `list` but see other comment. I'm not sure what the best way to represent a CSV download response is, I've never documented that kind of endpoint before.